spanner/spanner_snippets/spanner/spanner_cancel_backup_create.go

```diff
@@ -20,16 +20,23 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"regexp"
 	"time"
 
 	database "cloud.google.com/go/spanner/admin/database/apiv1"
+	pbt "github.com/golang/protobuf/ptypes/timestamp"
 	"google.golang.org/genproto/googleapis/longrunning"
 	adminpb "google.golang.org/genproto/googleapis/spanner/admin/database/v1"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
 func cancelBackup(w io.Writer, db, backupID string) error {
+	matches := regexp.MustCompile("^(.*)/databases/(.*)$").FindStringSubmatch(db)
+	if matches == nil || len(matches) != 3 {
+		return fmt.Errorf("Invalid database id %s", db)
```

Comment:
```suggestion
		return fmt.Errorf("invalid database id %q", db)
```

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Done, here and in the other two samples that are edited in this PR.